resources/pool.rb

```diff
@@ -30,6 +31,13 @@
     end
   end
 
+  current_enabled_status = f5.node_is_enabled?(new_resource.host)
+  if (new_resource.enabled == false && current_enabled_status == true)
+    f5.node_disable!(new_resource.host)
+  elsif (new_resource.enabled == true && current_enabled_status == false)
+    # enable
```

Comment:
So if I'm reading this right we will not re-enable a member that has been disabled by the admin outside of Chef? I think that's the sanest approach.

Reply:
Good point. This wasn't explicitly intended but it may be important to discuss.

There is a todo in the PR comments stating that I intend to add this re-enable as I envision the desired state to be controlled by node attributes but I'm new to this so guidance is welcome.